src/main/java/nextstep/ladder/domain/Person.java

```diff
@@ -7,8 +7,8 @@
   private String name;
 
   private Person(String name) {
-    if (name.length() > Ladder.MAX_NAME_LENGTH) {
-      throw new IllegalArgumentException("name length > 5.");
+    if (name.length() > 5) {
+      throw new IllegalArgumentException("name.length < 6");
```

Comment:
안내문구를 좀 변경하시면 추출하셨던 매직넘버를 잘 활용하실 수 있을 것 같습니다 😄 
예를 들자면, 
```suggestion
      if (MAX_LENGTH < name.length()) {
      throw new IllegalArgumentException("Name must not exceed " + MAX_LENGTH);
    }
```